src/ucp/core/ucp_worker.c

```diff
@@ -1068,6 +1068,7 @@ ucs_status_t ucp_worker_create(ucp_context_h context,
     worker->inprogress        = 0;
     worker->ep_config_max     = config_count;
     worker->ep_config_count   = 0;
+    worker->am_message_id     = 0;
```

Comment:
need to start from some random value (ucs_generate_uuid)

Reply:
updated